### PR TITLE
Fixed Checkbox radio shadow 20859

### PR DESCRIPTION
**Pull request by abrarpathan19**
### Fixed Issues (if relevant)

1. magento/magento2#20859 Luma theme - Input Box and Radio Button shadow is not proper 

### Manual testing scenarios (*)

1. Magento luma theme frontend in all the form
2. You can find checkbox, Radio input box shaddow
3. Reproduce pages : create account page, cart page and chekout page

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @abrarpathan19. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Review by VladimirZaets — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @VladimirZaets, thank you for the review.
[ENGCOM-4074](https://jira.corp.magento.com/browse/ENGCOM-4074) has been created to process this Pull Request

File: lib/web/css/source/lib/_resets.less [hunk 1]
```diff
@@ -105,6 +105,13 @@
             .lib-css(box-shadow, @focus__box-shadow);
         }
     }
+
+    input[type="radio"],
+    input[type="checkbox"] {
+        &:focus { 
+            .lib-css(box-shadow, none);
```
Comment (Karlasa) [inline lib/web/css/source/lib/_resets.less]: Hi! 
1) .lib-css(); wrapping is not needed as it has no variables inside to check
2) I would just change line 101 to instead  (just my opinion) 
`input:not([disabled]):not([type="radio"]):not([type="checkbox"])`
Cheers

**Review by Karlasa — CHANGES_REQUESTED** [review]

**Comment by Karlasa** [comment]
@VladimirZaets Could you double check please ;)

**Comment by abrarpathan19** [comment]
@VladimirZaets  & @Karlasa 
I have updated code as per the suggestion can you please review 
Thanks :)

**Review by Karlasa — APPROVED** [review]

**Comment by VladimirZaets** [comment]
@Karlasa @abrarpathan19  thanks!

**Pull request merged**